imap/structure.go

```diff
@@ -147,7 +146,7 @@ func getMIMEInfo(section *rfc822.Section) (string, string, map[string]string, er
 }
 
 func addDispInfo(c *paramList, writer parListWriter, header *rfc822.Header) {
-	if contentDisp, contentDispParams, err := mime.ParseMediaType(header.Get("Content-Disposition")); err == nil {
+	if contentDisp, contentDispParams, err := rfc822.ParseMediaType(header.Get("Content-Disposition")); err == nil {
```

Comment:
Should we remove the old `ParseMediaType` altogether now?

Reply:
The old one is provided by standard lib, there's nothing to remove